ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -232,6 +245,56 @@
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "amendChildrenGatekeeping",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "CRU"
```

Comment:
If they are only amending do we need create? Is read & update not enough?

Reply:
Should be enough let me change

Reply:
Ah no, it needs to be on the event I think? I mean we are always doing that like that, I can explore this separetely if you don't mind.

Reply:
Happy for this to be explored separately.